dsl/src/commonTest/kotlin/it/unibo/collektive/alignment/TestAlignment.kt

```diff
@@ -59,10 +64,16 @@ class TestAlignment : StringSpec({
         fun foo(value: Int, aggregate: Aggregate<Int>) = aggregate.neighboringViaExchange(value)
         acProgram { foo(this, x) } shouldNot alignWith { foo(x, this) }
     }
-    "Outer non-collective function taking collective function as argument should align the non-collective function" {
+    "Outer non-collektive function taking collective function as argument should align the non-collective function" {
```

Comment:
```suggestion

    "Outer non-collektive function taking collective function as argument should align the non-collective function" {
```

Eithers space all tests or none